.projenrc.js

```diff
@@ -1,15 +1,39 @@
-const { awscdk } = require('projen');
+const { awscdk, JsonPatch } = require('projen');
 const project = new awscdk.AwsCdkConstructLibrary({
-  author: 'Kaizen Conroy',
-  authorAddress: 'zen.conroy@gmail.com',
-  cdkVersion: '2.1.0',
+  author: 'Amazon Web Services',
+  cdkVersion: '2.0.0',
   defaultReleaseBranch: 'main',
-  name: 'awscdk-asset-kubectl',
+  name: '@aws-cdk/asset-kubectl-v1-20',
```

Comment:
Naming suggestions? `@aws-cdk/asset-kubectl-v1.20` does not work. I don't know if people care about the minor version of kubectl here, maybe just `@aws-cdk/asset-kubectl-v1` is sufficient?

Reply:
people do care about the minor version. Each minor version bump in kubernetes can have breaking changes. How about we borrow the convention from cdk8s-plus https://www.npmjs.com/package/cdk8s-plus-21

`@aws-cdk/asset-kubectl-21`

Reply:
Ok. Looks like this should be `@aws-cdk/asset-kubectil-v20` then.